device/pensando/arm64-elba-asic-flash128-r0/system_health_monitoring_config.json

```diff
@@ -0,0 +1,11 @@
+{
+    "services_to_ignore": [],
+    "devices_to_ignore": ["psu","fan"],
+    "user_defined_checkers": ["fetch_dpu_status"],
```

Comment:
@rameshraghupathy NPU is cable to know online/offline status of DPU. why we need this?

Reply:
@prgeor A binary state is not good enough to understand why a DPU is not behaving.  The DPU may be shown as online but a container may be down.  It is good indicator of such events.  This so helpful for us to debug already.